test/Microsoft.ML.Benchmarks.Tests/BenchmarksTest.cs

```diff
@@ -39,14 +42,25 @@ public class BenchmarksTest
 
         private ITestOutputHelper Output { get; }
 
-#if DEBUG
-        [Fact(Skip = SkipTheDebug)]
+#if DEBUG || NET462
+        [ConditionalTheory(Skip = SkipTheDebugAndFullFramework)]
```

Comment:
this could be simplified: we are using `if` and `ConditionalTheory` here while the `ConditionalTheory` could just point to a method encapsulating the logic. For example:

```cs
        public bool CanExecute =>
#if DEBUG
            false; // BenchmarkDotNet does not allow running the benchmarks in Debug, so this test is disabled for DEBUG
#elif NET462
            false; // We are currently not running Benchmarks for FullFramework
#else
            Environment.Is64BitProcess; // we don't support 32 bit yet
#endif

[ConditionalTheory(typeof(BenchmarksTest), nameof(CanExecute))]
```